libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -53,10 +53,12 @@ export class BookSearchComponent implements OnInit {
   }
 
   searchBooks() {
-    if (this.searchForm.value.term) {
-      this.store.dispatch(searchBooks({ term: this.searchTerm }));
-    } else {
-      this.store.dispatch(clearSearch());
-    }
+    setTimeout(()=>{
```

Comment:
This is wrong. DO NOT use `setTimeout`. 

Use form builder `valueChanges` Observable and RxJS operators for debounce.